CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Packaging
+
+- Minimum rust version was bumped to `1.41.0`
+
+### Removed
+
+- Ability to create a Wayland clipboard from Display type directly using `create_clipboard`
+- Example for Wayland clipboard
```

Comment:
```suggestion
```

I don't think we need to document this though, right? Seems like documentation doesn't really care about version bumps, unless there was some major docs rework?

Reply:
Ok, ok.